packages/office-ui-fabric-react/src/components/DatePicker/DatePicker.test.tsx

```diff
@@ -93,7 +93,7 @@ describe('DatePicker', () => {
     datePicker.unmount();
   });
 
-  it('should call onSelectDate only once when allowTextInput is true and popup is used to select the value', () => {
+  xit('should call onSelectDate only once when allowTextInput is true and popup is used to select the value', () => {
```

Comment:
use it.skip instead so it reports that this was skipped. If this is short-term, maybe add a `//todo`?

Reply:
Nevermind, xit is the same as jest.skip, I just always used jest.skip instead :)

Reply:
TIL and done.